client/src/components/AvailableDatatypes.vue

```diff
@@ -0,0 +1,83 @@
+<script setup>
```

Comment:
Overall this PR looks really nice. We should move this component-file into a subdirectory.

Reply:
What would you suggest calling this subdirectory? Or is there an exisitng one this might fit in?

Reply:
Up to you, but I see there is one that is called `Datatypes`, might be ok, although content seems to be be structured differently.